### ISSUE TITLE
Max FTW

### ISSUE THREAD

**Issue by ramprao** [mgrau/ad9959#2]
According to the datasheet for AD9959, the max FTW is `2**32`
 Your program AD9959.py uses `2**32-1`. Could you please clarify? Thanks.

**Comment by mgrau** [comment]
You are correct. While the phase accumulator is 32 bits, the allowed range for the FTW is 0 <= FTW <= 2^31. The DDS will only be able to produce tones that are up to half the system clock.
